analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -475,4 +476,29 @@ class SpdxDocumentFile(
 
         return listOf(ProjectAnalyzerResult(project, packages.toSortedSet()))
     }
+
+    /**
+     * Create the final [PackageManagerResult] by making sure that packages are removed from [projectResults] that
+     * are also referenced as project dependencies.
+     */
+    override fun createPackageManagerResult(
+        projectResults: Map<File, List<ProjectAnalyzerResult>>
+    ): PackageManagerResult {
+        val projectIds =
+            projectResults.flatMapTo(mutableSetOf()) { projResult -> projResult.value.map { it.project.id } }
```

Comment:
We usually don't use abbreviations like `projResult` but always use the full names like `projectResult`. Also the naming in this case is a bit misleading as it's actually a map entry, I would suggest to use:
`{ (_, projectResult) -> projectResult.map { it.project.id } }`